lib_eio/fs.ml

```diff
@@ -12,6 +12,10 @@ exception Already_exists of path * exn
 exception Not_found of path * exn
 exception Permission_denied of path * exn
 
+class virtual read_at_offset = object
```

Comment:
Might as well make a `ro` type for this and related operations. We already have `rw` type.
```suggestion
class virtual ro = object (_ : <Generic.t; Flow.source; ..>)
```

Reply:
Thanks, done.